src/util/commandline/option-decorators.ts

```diff
@@ -181,7 +181,7 @@ export function required(proto: any, propertyKey: string): void {
 
 // DefaultValue is also special, since it has to work with both as well. Same
 // basic logic
-export function defaultValue(value: string | number): PropertyDecorator {
+export function defaultValue(value: string): PropertyDecorator {
```

Comment:
Didn't you just add the `| number` in your last PR?

Reply:
@christav, yes, we've added the `| number` in the https://github.com/Microsoft/mobile-center-cli/pull/293
The main idea was to support `number` type for args in addition to `string` and `boolean`, but unfortunately it does not work like that.